project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_quickpost.scss

```diff
@@ -31,6 +31,10 @@
       right: 7px;
 
       label {
+        color: transparent;
```

Comment:
The label contains `<i>` tag that - attachment icon.
If we use `visibility: hidden`, it will be hidden.

`transparent color` is to hide label text, which is required for accessibility only.